src/containers/ec2.js

```diff
@@ -0,0 +1,44 @@
+const axios = require('axios');
+const utils = require('../utils');
+const eventIterface = require('../event');
+
+let currentEC2Labels = null;
+
+const URL = 'http://169.254.169.254/latest/meta-data/';
```

Comment:
why is it a static ip?

Reply:
it's always the same IP to get the metadata in the running instance